lib/5.26.2/x86_64-linux/Config_heavy.pl

```diff
@@ -1122,12 +1122,12 @@ sub myconfig {
 mallocobj=''
 mallocsrc=''
 malloctype='void *'
-man1dir='/home/git/binary-com/perl/man/man1'
-man1direxp='/home/git/binary-com/perl/man/man1'
-man1ext='1'
-man3dir='/home/git/binary-com/perl/man/man3'
-man3direxp='/home/git/binary-com/perl/man/man3'
-man3ext='3perl'
+man1dir=' '
```

Comment:
s/{SPACE}// ?

Reply:
Again another automatic `./Configure`/`make` change; we're due to update this anyway in a subsequent clean rebuild over a container.